docs/static/breaking-changes-pre63.asciidoc

```diff
@@ -3,6 +3,6 @@
 
 If you are upgrading from Logstash 6.2.x or any earlier version (including 5.x)
 and have the persistent queue enabled, we strongly recommend that you drain or
-delete the persistent queue before you upgrade. See <<upgrading-logstash-pqs>>
+delete the persistent queue before you upgrade. See <<draining-pqs>>
 for information and instructions.
```

Comment:
Note that this info is in Breaking Changes, not Upgrade.  If we decide to remove it post-release, we can do that.